inputfiles/idlSources.json

```diff
@@ -358,6 +362,10 @@
         "url": "https://w3c.github.io/mediacapture-main/",
         "title": "Media Capture and Streams"
     },
+    {
+        "url": "https://w3c.github.io/media-playback-quality/",
+        "title": "Media Playback Quality"
+    },
```

Comment:
This one seems totally fine, it's an editor's draft - yep, but every browser seems to be supporting it